astropy/coordinates/matrix_utilities.py

```diff
@@ -46,7 +46,7 @@ def rotation_matrix(angle, axis='z', unit=None):
     ----------
     angle : convertible to `~astropy.coordinates.Angle`
         The amount of rotation the matrices should represent.  Can be an array.
-    axis : str, or array-like
+    axis : str, or array_like
```

Comment:
Comma is unnecessary when there are only two items?
```suggestion
    axis : str or array_like
```